Cargo.toml

```diff
@@ -30,7 +30,7 @@ uuid = { version = "^1.4.1", features = [ "v4", "serde" ] }
 os-release = "^0.1.0"
 hostname = "^0.3.1"
 openssl = "^0.10.55"
-compact_jwt = { version = "0.3.4", optional = true }
+compact_jwt = { path = "../compact-jwt", optional = true }
```

Comment:
The changes in https://github.com/kanidm/compact-jwt/pull/25 need to be upstream before this can merge.